### PR TITLE
enh(Context): Add new properties

### PR DESCRIPTION
**Pull request by Thebarda**
This adds some new properties to user context and adds to new state : downtime and refresh interval

### CODE REVIEW

File: src/UserContext.ts [hunk 1]
```diff
@@ -27,9 +28,17 @@ const defaultAcl = {
   },
 };
 
-const defaultContext = {
+const defaultDowntime = {
+  default_duration: 3600,
```
Comment (kduret) [inline src/UserContext.ts:32]: Currently, default duration is 2 hours
```suggestion
  default_duration: 7200,
```

**Comment by bdauria** [comment]
@Thebarda the CI is not yet wired to this repo, so can you make sure that `npm run eslint` runs properly?

File: src/useDowntime.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { defaultDowntime } from './UserContext';
+import { Downtime } from './types';
```
Comment (bdauria) [inline src/useDowntime.ts:3]:
```suggestion
import * as React from 'react';

import { defaultDowntime } from './UserContext';
import { Downtime } from './types';
```

File: src/useRefreshInterval.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { defaultRefreshInterval } from './UserContext';
+import { Downtime } from './types';
```
Comment (bdauria) [inline src/useRefreshInterval.ts:3]:
```suggestion
import * as React from 'react';

import { defaultRefreshInterval } from './UserContext';
import { Downtime } from './types';
```

**Comment by Thebarda** [comment]
> @Thebarda the CI is not yet wired to this repo, so can you make sure that `npm run eslint` runs properly?

Now, `npm run eslint` runs correctly. I've updated `frontend-core`

**Review by bdauria — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**